src/main/java/com/team4/testingsystem/controllers/TestsController.java

```diff
@@ -0,0 +1,73 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.entities.Test;
+
+import com.team4.testingsystem.security.CustomUserDetails;
+import com.team4.testingsystem.services.TestsService;
+import com.team4.testingsystem.utils.jwt.JwtTokenUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+
+@RestController
+@RequestMapping(path="/tests")
+public class TestsController {
+
+    @Autowired
+    private TestsService testsService;
+
+    @Autowired
+    private JwtTokenUtil jwtTokenUtil;
+
+    private long getUserId() {
+
+ return ((CustomUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getId();
```

Comment:
Let's store it in the same file as the PR I've sent. Otherwise we will have lots of copies of the same code